site/content/docs/5.1/forms/checks-radios.md

```diff
@@ -102,24 +102,102 @@ Add the `disabled` attribute and the associated `<label>`s are automatically sty
 
 ## Switches
 
-A switch has the markup of a custom checkbox but uses the `.form-switch` class to render a toggle switch. Consider using `role="switch"` to more accurately convey the nature of the control to assistive technologies that support this role. In older assistive technologies, it will simply be announced as a regular checkbox as a fallback. Switches also support the `disabled` attribute.
+A switch has the markup of a custom checkbox but uses the `.form-switch`
+class to render a toggle switch. Consider using `role="switch"` to more
+accurately convey the nature of the control to assistive technologies that
+support this role. In older assistive technologies, it will simply be
+announced as a regular checkbox as a fallback. Switches also support the
+`disabled` attribute. They also can be theme-colored when they are checked
+and another when they are not checked.
 
 {{< example >}}
-<div class="form-check form-switch">
-  <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDefault">
-  <label class="form-check-label" for="flexSwitchCheckDefault">Default switch checkbox input</label>
+<div class="row fw-bold">
+  <div class="col">Default</div>
 </div>
-<div class="form-check form-switch">
-  <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckChecked" checked>
-  <label class="form-check-label" for="flexSwitchCheckChecked">Checked switch checkbox input</label>
+<div class="row">
+  <div class="col">
+    <div class="form-check form-switch">
+      <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDefault">
+      <label class="form-check-label" for="flexSwitchCheckDefault">Default switch checkbox input</label>
+    </div>
+  </div>
+  <div class="col">
+    <div class="form-check form-switch">
+      <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDisabled" disabled>
+      <label class="form-check-label" for="flexSwitchCheckDisabled">Disabled switch checkbox input</label>
+    </div>
+  </div>
 </div>
-<div class="form-check form-switch">
-  <input class="form-check-input" type="checkbox" role="switch" id="flexSwitchCheckDisabled" disabled>
-  <label class="form-check-label" for="flexSwitchCheckDisabled">Disabled switch checkbox input</label>
+
+<div class="row">
+  <div class="col fw-bold">Checked status</div>
+  <div class="col fw-bold">Not checked status</div>
+</div>
```

Comment:
Yes I tried but due to longer labels I find this solution more elegant to display. And the special case where swich-on and switch-off can be applied at the same time is a bit harder to display.

I will try again your solution.

Reply:
Here is another attempt but I am not super convinced: by reading/seeing this I don't really know which use cases could be covered. But this might not what the documentation is meant for, isn't it?
![image](https://user-images.githubusercontent.com/457663/153003890-39bd459e-7f11-47d1-aac1-880ee66bf0ff.png)